static/js/components/tiles/chart_tile.tsx

```diff
@@ -81,22 +81,37 @@ export function ChartTileContainer(props: ChartTileContainerProp): JSX.Element {
     const chartTitle = props.title
       ? formatString(props.title, props.replacementStrings)
       : "";
-    const svgElemList = containerRef.current.getElementsByTagName("svg");
-    const svgElem = svgElemList.length ? svgElemList.item(0) : null;
-    let svgXml = "";
-    let svgWidth = 0;
-    let svgHeight = 0;
-    if (svgElem) {
-      svgXml = svgElem.outerHTML;
-      const svgBBox = svgElem.getBBox();
-      svgWidth = svgBBox.width;
-      svgHeight = svgBBox.height;
+    const svgElemList = containerRef.current.getElementsByTagName("svg") as SVGElement[];
+    // Create new svg element to return which will hold all the svgs coming from
+    // containerRef
+    const embedSvg = document.createElementNS("http://www.w3.org/2000/svg", 'svg');
+    let embedSvgWidth = 0;
+    let embedSvgHeight = 0;
+    // Set embedSvgHeight as the max height of all svgs
+    Array.from(svgElemList).forEach((svg) => {
+      embedSvgHeight = Math.max(svg.getBBox().height, embedSvgHeight);
+    })
+    // Add each svg from svgElemList to embedSvg
+    for (const svg of svgElemList) {
+      const svgBBox = svg.getBBox();
+      const clonedSvg = svg.cloneNode(true) as SVGElement;
+      // Set height of current svg to the height of the embedSvg
+      clonedSvg.setAttribute("height", `${embedSvgHeight}`);
```

Comment:
would this reshape the original svg? Say two svg are placed vertically, would this still work?

Reply:
If the original chart had more than 1 svg, the svgs would be reshaped to be placed next to each other horizontally & stretched to the full height

Reply:
Once the svg is created, the path/line values are fixed, if just changing the height, would they still look good?

Reply:
Yes, because
- if an svg has a viewbox attribute, changing the height will cause the svg to shift according to the aspect ratio
- if no viewbox set & height increases (embedSvgHeight is the max height of all the svgs), there won't be any change to the path/line values